great_expectations/checkpoint/actions.py

```diff
@@ -213,9 +215,9 @@ class SlackNotificationAction(DataDocsAction):
 
     type: Literal["slack"] = "slack"
 
-    slack_webhook: Optional[str] = None
-    slack_token: Optional[str] = None
-    slack_channel: Optional[str] = None
+    slack_webhook: Optional[Union[ConfigStr, str]] = None
+    slack_token: Optional[Union[ConfigStr, str]] = None
+    slack_channel: Optional[Union[ConfigStr, str]] = None
```

Comment:
Should a channel be a secret str? Felt less likely than the rest but I think consistency makes sense here

Reply:
I would go ahead and keep it substituable, although not because it needs to be secret.
It adds flexibility without further complicating anything.